roles/ceph-osd/tasks/main.yml

```diff
@@ -37,8 +37,8 @@
 - name: include_tasks common.yml
   include_tasks: common.yml
 
-- name: include ceph_disk_cli_options_facts.yml
-  include_tasks: ceph_disk_cli_options_facts.yml
+- name: include container_options_facts.yml
+  include_tasks: container_options_facts.yml
 
 - name: include build_devices.yml
```

Comment:
build_devices is only used for non-collocated scenario so we should remove this include too and the associated file